scripts/migration-task-script.py

```diff
@@ -7,6 +8,30 @@
 environment = sys.argv[1]
 docker_image = sys.argv[2]
 
+# Check if there are migrations to apply
+try:
+    subprocess.getoutput("copilot svc exec --name post-award --command launcher flask db upgrade > current.txt")
+
+    result = subprocess.getoutput("copilot svc exec --name post-award --command 'grep head current.txt'")
+except subprocess.CalledProcessError as e:
+    sys.exit(e.returncode)
+
+# command returns extra lines besides the grep'd text
+match = re.search(r"\d+(?=\s\(head\))", result)
```

Comment:
This regex doesn't match our standardised alembic version name, which is `###_<some string>`. So the command outputs eg `038_current_fund_data (head)`